pkg/enclave_encrypt/encrypt.go

```diff
@@ -205,12 +206,12 @@ func decrypt(
 
 	receiver, err := suite.NewReceiver(receiverPrivate, []byte(TurnkeyHpkeInfo))
 	if err != nil {
-		return []byte{}, nil
+		return nil, fmt.Errorf("bad receiver private key")
```

Comment:
no wrap?

Reply:
oh right we should probably pass `err` in there!

Reply:
Addressed in https://github.com/tkhq/go-sdk/pull/51/commits/134726221b024dd8f6a3b534634135608b16747f